src/CMakeLists.txt

```diff
@@ -150,6 +150,8 @@ find_package(Threads REQUIRED)
 if(NOT CMAKE_USE_PTHREADS_INIT)
   message(FATAL_ERROR "POSIX threads: not found")
 endif()
+# Append thread flags/libraries to LIBS
+list(APPEND LIBS "${MAKE_THREAD_LIBS_INIT}")
```

Comment:
If you'd like, I could overwrite branch as a single commit not to pollute the project history with such commits.